designs.js

```diff
@@ -55,16 +51,16 @@ function selectRowHandler(event){
   selectedRowIndex = clickedRow.rowIndex;
 }
 
-function save(){
+function save(event){
```

Comment:
Could you please remove event from the argument list? It looks like we don't use it in the function.